apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js

```diff
@@ -37,6 +37,19 @@ $(document).ready(() => {
   init();
 
   function init() {
+    // TO-DELETE ONCE OPTIMIZELY-EXPERIMENT IS COMPLETE (start)
+    const queryString = window.location.search.substring(1).split('&');
+    const inVariant = queryString.includes('inSignUpUserTypeVariant=true');
```

Comment:
You can take a look at these PRs to see how they set and retrieve experiment flags.
https://github.com/code-dot-org/code-dot-org/pull/29335
https://github.com/code-dot-org/code-dot-org/pull/28653

Reply:
Thank you! I switched it to follow that format.

Reply:
Oh very cool! Thanks for the context, @hacodeorg !